public/bundle.js

```diff
@@ -106,7 +106,7 @@ eval("__webpack_require__.r(__webpack_exports__);\n/* harmony import */ var reac
 /***/ (function(module, __webpack_exports__, __webpack_require__) {
```

Comment:
gitignore your bundle.js -- you can just run react-dev, if you ever clone down and it'll build it for you